mne/io/brainvision/brainvision.py

```diff
@@ -318,8 +338,11 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
         Events from the corresponding vmrk file.
     """
     scale = float(scale)
-
+    vhdr_fname = os.path.abspath(vhdr_fname)
     ext = os.path.splitext(vhdr_fname)[-1]
+    if ext == '':  # lazy user gives filename with no extension
+        ext = '.vhdr'
+        vhdr_fname += ext
```

Comment:
Is it not possible to have a file with no extension? Maybe check ``if not op.exists(vhdr_fname)`` before adding the extension?